pages/watering.js

```diff
@@ -0,0 +1,59 @@
+import WateringSchedule from "@/components/WateringSchedule";
+import { useEffect, useState } from "react";
+import Image from "next/image";
+import BackArrow from "@/components/MyPlant/BackArrow";
+
+function useDayCount(startDay) {
+  const [day, setDay] = useState(startDay);
+  function handleNextDay() {
+    setDay(day + 1);
+  }
```

Comment:
Consider to use a functional update for the state to ensure the latest state value is always used -  safety first 👮

```suggestion
function handleNextDay() {
    setDay((currentDay) => currentDay + 1); 
      }
```